src/workerd/api/global-scope.c++

```diff
@@ -685,14 +685,26 @@ jsg::JsString ServiceWorkerGlobalScope::atob(jsg::Lock& js, kj::String data) {
 }
 
 void ServiceWorkerGlobalScope::queueMicrotask(jsg::Lock& js, v8::Local<v8::Function> task) {
-  // TODO(later): It currently does not appear as if v8 attaches the continuation embedder data
-  // to microtasks scheduled using EnqueueMicrotask, so we have to wrap in order to propagate
-  // the context to those. Once V8 is fixed to correctly associate continuation data with
-  // microtasks automatically, we can remove this workaround.
-  KJ_IF_SOME(context, jsg::AsyncContextFrame::current(js)) {
-    task = context.wrap(js, task);
-  }
```

Comment:
For review: Just noting... we no longer need to manually propagate the async context to `EnqueueMicrotask()` since v8 finally closed that gap recently. Just hadn't gotten around to removing this previously.